components/edit_post_modal/edit_post_modal.jsx

```diff
@@ -473,7 +476,9 @@ class EditPostModal extends React.PureComponent {
                         </div>
                     </div>
                 </Modal.Body>
-                <Modal.Footer>
+                <Modal.Footer
+                    style={{position: 'relative'}}
```

Comment:
I imagine this'll probably move if we move the emoji picker, but wherever it ends up, can you modify the CSS to set the position instead of styling it inline? The CSS for this one should be in `_modals.scss`